tracing/propagation/opencensus/propagation.go

```diff
@@ -0,0 +1,96 @@
+package opencensus
+
+import (
+	"context"
+	"encoding/hex"
+	"errors"
+	"fmt"
+
+	servicebus "github.com/Azure/azure-service-bus-go"
+	"go.opencensus.io/trace"
+)
+
+const (
+	TraceIdKey      = "go-shuttle/traceId"
+	SpanIdKey       = "go-shuttle/spanId"
+	TraceSampledKey = "go-shuttle/traceSampled"
+)
+
+func SpanContextFromMessage(msg *servicebus.Message) (trace.SpanContext, bool) {
```

Comment:
heavily inspired by existing http propagation

Reply:
Leave a liink to the http propogation inspiratioN?